front_end/entrypoints/rn_fusebox/rn_fusebox.ts

```diff
@@ -106,6 +106,18 @@ class FuseboxClientMetadataModel extends SDK.SDKModel.SDKModel<void> {
           const success = false;
           Host.rnPerfMetrics.fuseboxSetClientMetadataFinished(success, reason);
         });
+
+    if (!Common.Settings.Settings.instance().moduleSetting('preserve-console-log').get()) {
+      target.model(SDK.ConsoleModel.ConsoleModel)
+          ?.addMessage(new SDK.ConsoleModel.ConsoleMessage(
+              target.model(SDK.RuntimeModel.RuntimeModel), Protocol.Log.LogEntrySource.Recommendation,
+              Protocol.Log.LogEntryLevel.Info,
+              '[React Native] Console messages are currently cleared upon DevTools disconnection. You can preserve logs in settings: ',
```

Comment:
Text suggestions welcome!

Reply:
Do we support other languages besides English?

Reply:
Sorry missed this. Yes I should use an i18n string. Don't think we're currently doing any translation at the moment but good practice regardless. Will follow up